crates/p2p/src/client/conv.rs

```diff
@@ -656,13 +659,21 @@ impl TryFrom<(p2p_proto::receipt::Receipt, TransactionIndex)> for crate::client:
                         keccak: common.execution_resources.builtins.keccak.into(),
                         poseidon: common.execution_resources.builtins.poseidon.into(),
                         segment_arena: 0,
+                        add_mod: common.execution_resources.builtins.add_mod.into(),
+                        mul_mod: common.execution_resources.builtins.mul_mod.into(),
+                        range_check96: common.execution_resources.builtins.range_check96.into(),
                     },
                     n_steps: common.execution_resources.steps.into(),
                     n_memory_holes: common.execution_resources.memory_holes.into(),
-                    data_availability: ExecutionDataAvailability {
+                    data_availability: L1Gas {
                         l1_gas: GasPrice::try_from(common.execution_resources.l1_gas)?.0,
                         l1_data_gas: GasPrice::try_from(common.execution_resources.l1_data_gas)?.0,
                     },
+                    // TODO
```

Comment:
This still seems to be missing.

Reply:
We don't have proper fields in the p2p messages for now, so this property is not propagated via p2p.

Reply:
Anyway we should agree with the other node impls on adding some field in the proto because otherwise the receipt commitments will not match.

Reply:
I added those missing fields for the time being and raised a pr to the spec.